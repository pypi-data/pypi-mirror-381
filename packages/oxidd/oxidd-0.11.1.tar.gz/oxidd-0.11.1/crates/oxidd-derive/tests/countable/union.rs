use oxidd_derive::Countable;

#[derive(Countable)]
union Tag {
    f1: u8,
    f2: u32,
}

fn main() {}
