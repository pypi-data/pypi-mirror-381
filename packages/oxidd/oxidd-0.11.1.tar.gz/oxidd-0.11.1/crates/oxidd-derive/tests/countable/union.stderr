error: `Countable` cannot be derived for `union`s
 --> tests/countable/union.rs:4:1
  |
4 | union Tag {
  | ^^^^^
